auto-dark.el

```diff
@@ -26,6 +26,10 @@
 
 
 ;;; Code:
+
+;; Optional require of dbus squelches elisp compilation warnings
+(require 'dbus nil t)
```

Comment:
This require will fail silently if Emacs was compiled without the `dbus` library.  I only put it here to squelch compilation warnings, as noted.